services/system/Accounts/plugin/src/lib.rs

```diff
@@ -29,6 +43,39 @@ impl Accounts for AccountsPlugin {
         Err(NotYetImplemented.err("add_auth_service"))
     }
 
+    fn get_account(name: String) -> Result<Option<AccountTypes::Account>, CommonTypes::Error> {
+        let query = format!(
+            "query {{ getAccount(account: \"{}\") {{ accountNum, authService, resourceBalance }} }}",
+            AccountNumber::from(name.as_str())
+        );
+
+        let account_result = Server::post_graphql_get_json(&query)
+            .map_err(|e| QueryError.err(&e.message))
+            .and_then(|result| {
+                serde_json::from_str(&result).map_err(|e| QueryError.err(&e.to_string()))
+            });
+
+        let account = account_result.and_then(|response_root: ResponseRoot| {
+            response_root
+                .data
+                .getAccount
+                .ok_or_else(|| InvalidAccountNumber.err(&name))
```

Comment:
Error should be `AccountDNE` rather than `InvalidAccountNumber`. Just because it doesn't exist doesn't mean it's an invalid account number.